libs/localization/README.md

```diff
@@ -189,3 +197,13 @@ Add the `extract-strings` script to `workspace.json`. Running this script will e
   }
 }
 ```
+
+Running `yarn nx run your-project:extract-strings` will extract messages from the project and create or update a Namespace entry in Contentful, if the namespace did not exist it will need to be published in Contentful before the client can query the entry.
+
+**Notes**:
+
+- Adding a new id automatically creates it in Contentful.
+- Removing an id from the project will automatically remove that id from the Contentful Namespace entry.
```

Comment:
At the moment it doesn't remove it from the Contentful entry but marks it as deprecated. If the id is used again, it will be unmarked from being deprecated

Reply:
...added that detail as a footnote.